test/e2e/nodenetworkstate_test.go

```diff
@@ -213,10 +210,9 @@ var _ = Describe("NodeNetworkState", func() {
 			})
 			It("should have the bond in the linux bridge as port at currentState", func() {
 				var (
-					expectedInterfaces  = interfaces(br1WithBond1Up)
-					expectedBond        = interfaceByName(expectedInterfaces, "bond1")
-					expectedBridge      = interfaceByName(expectedInterfaces, "br1")
-					expectedBridgePorts = expectedBridge["bridge"].(map[string]interface{})["port"]
```

Comment:
why is this dropped? just a cleanup?

Reply:
expectedBridgePorts is not needed we are now harcoding "bond1" as the port name, if you don't then you have to do a partial search of attribute "name" in the port since we don't have the stp attribute in the expected data.